### PR TITLE
Allow auto dark mode and forcing theme

### LINKED ISSUE #151: Theme ignored after upgrading to v1
I've been using your theme for a few years, and I really like it. Thanks for publishing it.

Unfortunately, it seems to have broken for me after your recent changes upgrading to v1. When I push to my repository, the deployment to GitHub pages fails.

I tried to follow your instructions to upgrade, and I got it to deploy the site, but now it seems to have no theme at all!

Here's my `config.yml`:
```
remote_theme: chrisrhymes/bulma-clean-theme
title: Book Blender
google_analytics: UA-4644630-2
baseurl: /book-blender

defaults:
  - scope:
      path: ""
    values:
      hero_image: /book-blender/images/index_hero.jpg
      hero_darken: true
      twitter:
        card: summary_large_image
      image: images/index_hero.jpg
```

Here's my Gemfile:
```
source 'https://rubygems.org'
gem 'jekyll'
```

You can see the project at [`donkirkby/book-blender`].

I thought maybe I shouldn't be using it as a remote theme but as a gem theme, but that didn't show up either.

It's unfortunate that this breaks projects. Is there a way for projects to continue using the old version until they want to upgrade? Is there a way to learn about upcoming changes to your theme?

Let me know if I can help with documentation or some coding. I contributed a few small patches back in 2020.

[`donkirkby/book-blender`]: https://github.com/donkirkby/book-blender/tree/main/docs

### LINKED ISSUE #152: Bug: Not fixing the navbar forces light mode // Feature request: Use automatic dark mode per default, add option to enforce a specific theme
The following code in `/_layouts/default.html` defines the `class` attribute twice:
```
    {% if site.fixed_navbar %}
        class="has-navbar-fixed-{{ site.fixed_navbar }}"
    {% endif %}
    class="theme-light"
```

Firefox ignores the second definition. This leads to the following problem:
- When `fixed_navbar` is set to true, bulma.io's automatic dark mode works correctly, because `class` is defined twice and `class="theme-light"` is ignored by Firefox.
- When `fixed_navbar` is set to false, bulma.io's automatic dark mode does not work, because `class="theme-light"` overrides it.

Proposed solution:
- Only define `class` once
- Make it possible to enforce a dark or light theme by introducing `site.force_theme`

```
    class="{% if site.fixed_navbar %} has-navbar-fixed-{{ site.fixed_navbar }} {% endif %} {% if site.force_theme %} theme-{{ site.force_theme }} {% endif %}"
```

### PR DESCRIPTION
**Pull request by chrisrhymes**
Resolves #151, resolves #152 

- Fixes bug with fixed navbar and light theme being forced. 
- Allows auto dark mode detection or forcing a theme
- Add additional information to upgrade guide and improved docs based on feedback

### CODE REVIEW

File: _includes/notification.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <div x-data="{visible: true}" markdown="0">
-    <div class="notification {{ include.status | default: 'is-warning' }}" x-show.transition.duration.300ms="visible">
+    <div
```
Comment (chrisrhymes) [inline _includes/notification.html:2]: Add margin to top and bottom of notifications

**Comment by GruberMarkus** [comment]
I kindly ask for the following change to #153: Don't make `force_theme: light` the default, but let bulma.io decide if light or dark mode should be used based on the user's system and browser settings (`#force_theme: light`).

File: docs/getting-started/theming.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ show_sidebar: false
 
 Bulma Clean Theme uses the Bulma frontend framework. Check out the [Bulma docs](https://bulma.io/documentation/) for more information.
 
+## Bulma Themes
+
+Bulma v1 has the concept of [themes](https://bulma.io/documentation/features/themes/) and auto detects the browsers choice for dark or light theme.
```
Comment (donkirkby) [inline docs/getting-started/theming.md:16]:
```suggestion
Bulma v1 has the concept of [themes](https://bulma.io/documentation/features/themes/) and auto detects the browser's choice for dark or light theme. 
```

File: README.md [hunk 3]
```diff
@@ -46,6 +39,25 @@ Or install it yourself as:
...
+
+### v0.x
+
+If you are deploying to GitHub pages using their default build process, then you can install v0.x of the [GitHub Pages gem](https://github.com/github/pages-gem) and use `remote_theme` instead of `theme` in your `_config.yml`. 
+
+**Note that the GitHub Pages gem requires Jekyll version 3.9 and version 0.x of Bulma Clean Theme**
+
+```yaml
+# With GitHub Pages Gem
+remote_theme: chrisrhymes/bulma-clean-theme:v0.14.0
```
Comment (donkirkby) [inline README.md:56]: This is much clearer, thank you! I didn't know that you could pin the version of a remote theme. That looks like a simple solution to start with.

File: docs/getting-started/upgrading-to-v1.md [hunk 4]
```diff
@@ -10,12 +10,56 @@ toc: true
...
+$ bundle update
+```
+
+## Using remote_theme
+
+If you are using Jekyll Remote Theme, then you can [add a version number](https://github.com/benbalter/jekyll-remote-theme?tab=readme-ov-file#declaring-your-theme) in your _config.yml to specify which version you want to use.
+
+```yaml
+# _config.yml
+remote_theme: chrisrhymes/bulma-clean-theme@v1.0.0
```
Comment (donkirkby) [inline docs/getting-started/upgrading-to-v1.md:35]: Would you like this to be consistent with the README and installation pages? Both worked in my testing, so this isn't wrong.

```suggestion
remote_theme: chrisrhymes/bulma-clean-theme:v1.0.0
```

**Comment by chrisrhymes** [comment]
> I kindly ask for the following change to #153: Don't make `force_theme: light` the default, but let bulma.io decide if light or dark mode should be used based on the user's system and browser settings (`#force_theme: light`).

Thanks for your feedback. I have been thinking about this and decided to keep it as force_light by default as this is similar behaviour to how the previous version looked. You are free to set whatever value you like in your site's own _config.yml.

**Pull request merged**